src/lxml/tests/test_external_document.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+"""
+Test cases related to direct loading of external libxml2 documents
+"""
+
+import unittest
+import sys
+
+from common_imports import etree, HelperTestCase
+
+
+@unittest.skipIf(sys.version_info[:2] < (2, 7),
```

Comment:
`unittest.skipIf()` requires Python 2.7, though ;)
But there's a fallback for it in `common_imports`.